packages/backend/src/accounting/psql/ledger-transfer/index.ts

```diff
@@ -232,6 +227,10 @@ async function validateTransfer(
     return TransferError.DifferentAssets
   }
 
+  if (await LedgerTransfer.query(trx).findOne({ transferRef })) {
+    return TransferError.TransferExists
+  }
```

Comment:
Making two identical withdrawals/transfers with the second one putting an account into overbalance would result in a "insufficient balance error" instead of a "transfer exists error". This would cause the liquidity resolver tests to fail. It does feel "tidier" to skip balance checks if a transfer happens to exist, but does add one additional lookup.